torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from torch.fx.graph_module import GraphModule
+from torch.fx.graph import map_arg
+
+
+def get_all_users_of(index: int, fx_module: GraphModule) -> List[int]:
+    "Given the graph(fx_module) and an index, return a list of all node indexes that use this node"
+    graph = fx_module.graph
+    node = graph.nodes[index]
+    user_indexes: List[int] = []
+    """if the node A is in node B's args, then B is the user of A
+       go through all the nodes, if the input node in any node's args,
+       then that node is the input node's user
+    """
+    for index, n in enumerate(graph.nodes):
```

Comment:
`index` here shadows the function parameter, which can be confusing. I suggest using something like `i` instead